### DISCUSSION TITLE
Wallet sync stucked at 61965, but Full Node Status is Synced, how do i do? 钱包同步卡住不动了，该咋整？ 对收益有影响吗？

### DISCUSSION THREAD

**Discussion by wuyifan66** [Chia-Network/chia-blockchain#3212] [Q&A]
![stuked](https://app.hnust.edu.cn/static/cdn/QQ20210430195712.png)
![synced](https://app.hnust.edu.cn/static/cdn/QQ20210430200036.png)
As the screenshot show, Wallet sync is at 61965 and It has been stucked for several hours, I try to reboot the Chia and the computer, still not work.

So, how can i do now? 
And, any influence on my framing? Is that means if the wallet sync is not fininshed , there is no  Rewards i can get?

Chinese:
钱包同步卡住不动了，该咋整？ 对收益有影响吗？

**Comment by keliew** [comment]
Wallet not syncing is not related to and will not affect your farming abilities.
